lib/onigumo.ex

```diff
@@ -3,23 +3,29 @@ defmodule Onigumo do
   Web scraper
   """
   @input_filename "urls.txt"
+  def input_filename, do: @input_filename
   @output_filename "body.html"
+  def output_filename, do: @output_filename
 
   def main() do
     HTTPoison.start()
+    save_URLs_contents(@input_filename, @output_filename)
+  end
+
+  def save_URLs_contents(input_file, output_file) do
     http = http_client()
 
-    load_urls(@input_filename)
-    |> Enum.map(&download(http, &1))
+    load_urls(input_file)
```

Comment:
See, here we originally used lowercase _urls_.

Reply:
right díš